app/data_definitions/from_cascade/ad_landing.xml

```diff
@@ -0,0 +1,130 @@
+<system-data-structure>
```

Comment:
Copied directly from Dev Cascade. Now under version control. However, any changes in Cascade have to be copied over. So a bit of a double accounting issue.